.browserslistrc

```diff
@@ -0,0 +1,10 @@
+{
+  "production": [
+    "> 1%",
+    "ie 10"
+  ],
+  "development": [
+    "last 1 chrome version",
+    "last 1 firefox version"
+  ]
+}
```

Comment:
the syntax for `.browserslistsrc is different from `package.json` one. see https://www.npmjs.com/package/browserslist#environments and update the PR.